Code/Tools/ProjectManager/tests/PythonBindingsTests.cpp

```diff
@@ -53,13 +54,19 @@ namespace O3DE::ProjectManager
 
         void SetUp() override
         {
+            m_previousDirectInstance = AZ::IO::FileIOBase::GetDirectInstance();
+
+            AZ::IO::FileIOBase::SetDirectInstance(&m_fileIO);
```

Comment:
The dependency of the Project Manager test on the FileIOBase can be removed, by changing the use of `AZ::IO::FileIOBase::CreatePath` to be `AZ::IO::SystemFile::CreateDir`.

Reply:
(as mentioned earlier) PythonLoader::ReadPythonEggLinkPaths uses the DirectInstance version for FindFiles, which supports recursive searches where SystemFile::FindFiles does not.